plugin.yml

```diff
@@ -9,6 +9,10 @@ configuration:
       description: |
         The file(s) to upload; defaults to `dist/coverage/**/*.xml`.
       type: string
+    flags:
+      description: |
+        Flag the upload to group coverage metrics
+      type: string
```

Comment:
not a must-change:
"Flag the uploaded report in order to group it based on type of tests or sub-projects/teams."

also codecov uses the term 'report' over 'metrics' :)

Reply:
This one came straight from the `codecov --help` docs, but I've also updated to your suggestion

Reply:
oh, in that case codecov cli should make their help more helpful 😈